docs/default-directory-layout.md

```diff
@@ -62,6 +63,9 @@ Contains files not consumed by *any* process. Only for the repo as reference ass
 ### /test
 Contains tests for the cmdlets. By default, it contains an example pester test and a httpipelinemock.ps1 script.
 
+### /UX
+Contains metadata files for portal UX. Portal use these to display the usage of cmdlet for the resource in portal page.
```

Comment:
Support for UX metadata is **azure** specific. Please add description to explain it here and other related places. And another change needed is to only generate meta data when the **azure** is specified.